### PR TITLE
Translation of readme to English

### PR DESCRIPTION
**Pull request by miguelcarreiro**
readme.md - English version
readme.pt.md - Portuguese version

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,66 +1,66 @@
...
-- o acf poderia-se por a carregar por json, seria ligeiramente mais eficiente
-- edição do menu em apresentação > menus
-- edicao das colunas do footer em apresentação > widgets
-- Duplicar os posts para EN (e informação)
-- no functions.php tem um snippet a remover funções para editores
-- ao fazer upload não esquecer de remover o ambiente de desenvolvimento
-- e editar o wp-config com os dados correctos de ligação à BD.
+## Small notes
 
+- Css files have cache buster (no header.php);
```
Comment (quetzyg) [inline readme.md]: - A cache buster is enabled for CSS files (in the `header.php` file);

File: readme.md [hunk 2]
```diff
@@ -1,66 +1,66 @@
...
-Actualmente versão 5.1.1 do wp, com plugins wpml, acf, wp bakery, classic editor e acf extended forms instalados.
+Read this in [Portuguese](readme.pt.md)
 
- - **wpml** - gestão multilingua
- - **acf** - para criar custom fields para backoffice
-  - **acf** extended form - criar forms com o acf
-  - **classic editor** - para o editor funcionar (s/ guttenberg)
-  - **wp bakery** - para a criação de posts com templates personalizados.
-- tema criado de raiz para a VOST /vost
+Actual version: wp 5.1.1, com plugins wpml, acf, wp bakery, classic editor e acf extended forms instalados.
```
Comment (quetzyg) [inline readme.md]: `Current version: WordPress 5.1.1, with the following plugins installed: WPML, ACF, WPBakery, Classic Editor and ACF extended forms.`

File: readme.md [hunk 3]
```diff
@@ -1,66 +1,66 @@
...
+ - **[ACF](https://www.advancedcustomfields.com/)** _extended form_ - create forms with acf
+ - **[Classic Editor](https://wordpress.org/plugins/classic-editor/)** - restore previous WordPress editor (w/ guttenberg)
+ - **[WP Bakery](https://wpbakery.com/)** - create posts with custom templates
+ - Custom-made theme for VOST PT
 
-O ambiente de produção está feito para trabalhar com gulp, sass, babel e optimização de imagens. Pode-se trabalhar sem isto, mas os ficheiros de produção estão minificados.
+## Development environment
 
-encontra-se em */wp-content/themes/vost/dev*
+The production environment is made to work with GULP, SASS, Babel, and image optimization. You can work without this, but the production files are minified.
```
Comment (quetzyg) [inline readme.md]: `GULP` -> `gulp`

File: readme.md [hunk 4]
```diff
@@ -1,66 +1,66 @@
...
-- 
+## TODO
 
+- htaccess cache for imagens/static;
+- Content (and contact mails in form);
+- Translations;
+- Mailchimp integrated newsletter;
+- Archive page;
+- Global header/footer info, to make less calls to database;
+- Disable jquery global loading, and keep only in home for slickjs (try to optimize as much as possible).
```
Comment (quetzyg) [inline readme.md]: `jquery` -> `jQuery`

File: readme.md [hunk 5]
```diff
@@ -1,66 +1,66 @@
...
-- conteudo (e mails de contacto no form)
-- traduções
-- newsletter integrada com mailchimp
-- pagina de arquivo
-- info footer/header global para fazer menos calls a bd
-- remover jquery de carregar globalmente e apenas na home para o slickjs (tentar ter o mais optimizado possivel)
-- 
+## TODO
 
+- htaccess cache for imagens/static;
```
Comment (quetzyg) [inline readme.md]: `htaccess` -> `.htaccess`

File: readme.md [hunk 6]
```diff
@@ -1,66 +1,66 @@
...
 
+- Css files have cache buster (no header.php);
+- If, for some reason wpml is disabled, you must pay attention all called instances;
+- The same for acf, but you would have noticed :) ;
+- jquery is only needed for slickjs in home (optimization tip - call script only in home, it's already called in functions);
+- In order to improve efficiency, acf can be loaded using json, it would be more efficient;
+- Menu edition in apresentação > menus;
+- Footer columns edition in apresentação > widgets;
+- Duplicate posts to English (and info);
+- In functions.php exists a snippet that removes functions for editors;
```
Comment (quetzyg) [inline readme.md]: - There's a code snippet in the `functions.php` file that disables editor functionality;

**(not sure what the original sentence meant, I assume the above)**

File: readme.md [hunk 7]
```diff
@@ -1,66 +1,66 @@
...
-- Duplicar os posts para EN (e informação)
-- no functions.php tem um snippet a remover funções para editores
-- ao fazer upload não esquecer de remover o ambiente de desenvolvimento
-- e editar o wp-config com os dados correctos de ligação à BD.
+## Small notes
 
+- Css files have cache buster (no header.php);
+- If, for some reason wpml is disabled, you must pay attention all called instances;
+- The same for acf, but you would have noticed :) ;
+- jquery is only needed for slickjs in home (optimization tip - call script only in home, it's already called in functions);
```
Comment (quetzyg) [inline readme.md]: `jquery` -> `jQuery`

**Review by quetzyg — COMMENTED** [review]
Left some comments

**Comment by miguelcarreiro** [comment]
> Left some comments

@quetzyg Thanks for your feedback! I've corrected readme with all your suggestions, and will submit a new commit soon.

About your comment referring to line 53, I'm not the original author, so I've tried to translate this readme without changing the meaning of the phrases, but since I agreed with you, this note was also modified.

**Pull request merged**